pulsar-broker/src/test/java/org/apache/pulsar/broker/admin/AdminApiTest2.java

```diff
@@ -2231,4 +2232,170 @@ public void testFailedUpdatePartitionedTopic() throws Exception {
         // validate subscription is created for new partition.
         assertNotNull(admin.topics().getStats(partitionedTopicName + "-partition-" + 6).getSubscriptions().get(subName1));
     }
+
+    @Test
+    public void testPartitionedStatsAggregationByProducerName() throws Exception {
+        final String topic = "persistent://prop-xyz/ns1/test-partitioned-stats-aggregation-by-producer-name";
+        admin.topics().createPartitionedTopic(topic, 10);
+
+        @Cleanup
+        PulsarClient client = PulsarClient.builder().serviceUrl(pulsar.getWebServiceAddress()).build();
```

Comment:
Is it not possible to use `pulsarClient` declared in the super class?
https://github.com/apache/pulsar/blob/6382be84ae023edeae1ccc996d70391e8b01aec4/pulsar-broker/src/test/java/org/apache/pulsar/broker/auth/MockedPulsarServiceBaseTest.java#L87